CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 1.0.1 / 2019-12-21
+
+[ENHANCEMENT] Remove excessive whitespace from HTML templates. #302
```

Comment:
Is this even worth a mention? It's not really user visible.

Reply:
I was on the fence here, but since the cognitive load of a two-line changelog is still quite limited, I thought I mention it, just in case somebody is confused that suddenly the response size drops on the status page, or somebody was annoyed before about all the empty lines and tabs when inspecting the HTML code and has now a reason to rejoice over the holidays.